java/main/com/google/privacy/differentialprivacy/LaplaceNoise.java

```diff
@@ -149,17 +150,58 @@ public ConfidenceInterval computeConfidenceInterval(
       long lInfSensitivity,
       double epsilon,
       @Nullable Double delta,
-      double confidenceLevel) {
-    // TODO: Implement confidence interval computation.
-    throw new UnsupportedOperationException("Not implemented yet.");
+      double alpha) {
+    checkConfidenceParameters(l0Sensitivity, lInfSensitivity, epsilon, delta, alpha);
+    ConfidenceInterval confInt =
+        computeConfidenceInterval(
+            noisedX, Noise.getL1Sensitivity(l0Sensitivity, lInfSensitivity) / epsilon, alpha);
+    return ConfidenceInterval.create(
+        Math.round(confInt.lowerBound()), Math.round(confInt.upperBound()));
   }
 
   private void checkParameters(double l1Sensitivity, double epsilon, @Nullable Double delta) {
-    DpPreconditions.checkEpsilon(epsilon);
+    DpPreconditions.checkEpsilonVeryStrict(epsilon);
     DpPreconditions.checkNoiseDelta(delta, this);
     DpPreconditions.checkL1Sensitivity(l1Sensitivity);
   }
 
+  private void checkConfidenceParameters(
+      int l0Sensitivity,
+      double lInfSensitivity,
+      double epsilon,
+      @Nullable Double delta,
+      double alpha) {
+    DpPreconditions.checkEpsilonStrict(epsilon);
+    DpPreconditions.checkNoiseDelta(delta, this);
+    DpPreconditions.checkSensitivities(l0Sensitivity, lInfSensitivity);
+    DpPreconditions.checkAlpha(alpha);
+  }
+
+  /**
```

Comment:
nit: ideally, the style of this doc is consistent with that of Gaussian noise. I made a suggestion in the Gaussian noise version that is a bit more succinct and consistent with the addNoise doc. 

other nit: "{@code lambda}" rather than "lambda"